### PR TITLE
✨ Add Database connection for RegisterPage

### PR DESCRIPTION
**Pull request by Dev-dfm**

### CODE REVIEW

File: src/app/pages/Register/Register.tsx [hunk 1]
```diff
@@ -4,15 +4,32 @@ import BackButton from '../../components/BackButton/BackButton';
...
 import LabeledInput from '../../components/LabeledInput/LabeledInput';
 import Button from '../../components/Button/Button';
+import { User } from '../../../types';
+import { postUser } from '../../utils/api';
+import { useHistory } from 'react-router-dom';
 
 export default function Register(): JSX.Element {
+  const history = useHistory();
   const [userName, setUserName] = useState('');
   const [email, setEmail] = useState('');
```
Comment (franzosterkamp) [inline src/app/pages/Register/Register.tsx:14]: is can see that you are on a really good way 👍 

maybe you can think about adding a regEx pattern for your email and password validation :) 

for e.g. like this : https://www.tutorialspoint.com/regex-in-reactjs

but try to understand, what is happening there ->  https://regexr.com/

**Review by J-Bannach — APPROVED** [review]
lgtm

**Review by lajanzen — APPROVED** [review]
🚀

**Review by l-goede — COMMENTED** [review]
lgtm!:)

**Pull request merged**